src/main/com/fulcrologic/fulcro/routing/dynamic_routing.cljc

```diff
@@ -346,6 +366,9 @@
   ([this-or-app relative-class-or-instance new-route]
    (change-route-relative this-or-app relative-class-or-instance new-route {}))
   ([app-or-comp relative-class-or-instance new-route timeouts]
+ ;; TODO: Add next lines
+ ;;   (when-not (seq (proposed-new-path this-or-reconciler relative-class-or-instance new-route))
```

Comment:
Just ported the `this-or-reconciler` to newer `this-or-app`.

Reply:
If you look at the other arity of the function I chose the name app-or-comp, so keep consistent